<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterSettingDialog</class>
 <widget class="QDialog" name="ParameterSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>パラメータ設定</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>75</y>
     <width>226</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="3">
     <widget class="QLineEdit" name="LE_LastTimer0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LE_FirstTimer0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="Label_Timer0">
      <property name="text">
       <string>Timer0の範囲</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="Label_Timer0Range">
      <property name="text">
       <string>〜</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>322</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="Label_MacAddr">
      <property name="text">
       <string>MACアドレス</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LE_MacAddr1">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="LE_MacAddr2">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="LE_MacAddr3">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="LE_MacAddr4">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="LE_MacAddr5">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLineEdit" name="LE_MacAddr6">
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>443</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <widget class="QLabel" name="Label_Nazo">
      <property name="text">
       <string>nazo値</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="LE_Nazo1">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="LE_Nazo2">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="LE_Nazo3">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="LE_Nazo4">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLineEdit" name="LE_Nazo5">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>105</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QLineEdit" name="LE_VCount">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="Label_VCount">
        <property name="text">
         <string>VCount</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_GxStat">
        <property name="text">
         <string>GxStat</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LE_GxStat">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="Label_Frame">
        <property name="text">
         <string>Frame</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="LE_Frame">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="BTN_OK">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>86</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>LE_Nazo1</tabstop>
  <tabstop>LE_Nazo2</tabstop>
  <tabstop>LE_Nazo3</tabstop>
  <tabstop>LE_Nazo4</tabstop>
  <tabstop>LE_Nazo5</tabstop>
  <tabstop>LE_MacAddr1</tabstop>
  <tabstop>LE_MacAddr2</tabstop>
  <tabstop>LE_MacAddr3</tabstop>
  <tabstop>LE_MacAddr4</tabstop>
  <tabstop>LE_MacAddr5</tabstop>
  <tabstop>LE_MacAddr6</tabstop>
  <tabstop>LE_FirstTimer0</tabstop>
  <tabstop>LE_LastTimer0</tabstop>
  <tabstop>LE_VCount</tabstop>
  <tabstop>LE_GxStat</tabstop>
  <tabstop>LE_Frame</tabstop>
  <tabstop>BTN_OK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
